src/snowflake/snowpark/_internal/server_connection.py

```diff
@@ -656,7 +656,7 @@ def get_result_and_metadata(
 
     def get_result_query_id(self, plan: SnowflakePlan, **kwargs) -> str:
         # get the iterator such that the data is not fetched
-        result_set, _ = self.get_result_set(plan, to_iter=True, **kwargs)
+        result_set, _ = self.get_result_set(plan, ignore_results=True, **kwargs)
```

Comment:
this might be in another PR

Reply:
yeah. this was a mistake. thanks for catching it